tasks/extract_filepaths_for_granule/schemas/config.json

```diff
@@ -57,6 +57,14 @@
             ]
           }
         ]
+      },
+      "excludeFileTypes": {
```

Comment:
Should this be `excludeFileTypes` or `exclude_file_types`. I would try to make it consistent across the lambdas that use this config items.